### ISSUE TITLE
Export data to CSV or other Format

### ISSUE THREAD

**Issue by Wegi** [Phaetec/pogo-cruncher#61]

**Comment by MunchnetNL** [comment]
Would it be possible to have capture date included in the export ? 
It doesn't make much sense to have it show on page but I do like it. 
Usually I capture in the evening, next morning I check how many / what I captured. 
I use this info to make an overview for myself what routes I took are best for mass evolve vs capturing certain pokemon. 
Sinds I started doing this I found out that what I thought where efficient areas where less efficient then I thought. 
Having the date means I don't have to cross reference what I already had vs what's new.  

It's not critical to have but rather just a nice addition. 
Therefore I don't think it needs to be shown on the details on the page view.

**Comment by Wegi** [comment]
Yeah capture date is possible and doesn't mak to much work. Will include.

**Comment by sayya112** [comment]
was this implemented or is it hidden somewhere?

**Comment by n2o** [comment]
Nope, not implemented yet. It is an open issue
